LedgerHQ/ledger-live-wagmi-connector
# Update deps

fixes: #6 

- Update `wagmi` dependency to `^0.8.8` due to multiple breaking changes since previously used version `0.5.x`
- Update version of most `devDependencies`
- Rework the build system and tooling:
  - remove use of `tsdx`
  - replace `jest` by `vitest`
  - add `prettier`
  - add `changesets`
  - add `turbo` (used for `release` job)
  - don't build the package as `commonjs` anymore (since `wagmi` [does not support it](https://github.com/wagmi-dev/wagmi/blob/main/packages/core/package.json#L26))


📄  Note: the `size` CI job fails because it tries to build the lib on the `main` branch but the `@wagmi/core` dependency is missing on this commit (cf. [here](https://github.com/LedgerHQ/ledger-live-wagmi-connector/blob/main/package.json#L61-L65)).
This should be non blocking and will be taken care of if not resolved after merged.
